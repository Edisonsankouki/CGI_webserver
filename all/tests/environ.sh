cd ..
python3 webserv.py config.cfg &
PID=$!
cd -
sleep 1
str=$(curl localhost:8070/environ.py | diff -w - environ.out)
if [[ "$str" == "" ]];then
    echo test_environ passed
else
    echo test_environ failed
fi 
kill $PID