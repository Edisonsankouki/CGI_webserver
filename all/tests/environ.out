<h1>127.0.0.1</h1>
<h1>Accept: */*</h1>